powerline-themes.el

```diff
@@ -50,10 +50,10 @@
                                        (powerline-buffer-size mode-line 'l))
                                      (when powerline-display-mule-info
                                        (powerline-raw mode-line-mule-info mode-line 'l))
-                                     (powerline-buffer-id mode-line-buffer-id 'l)
+                                     (powerline-buffer-id mode-line 'l)
                                      (when (and (boundp 'which-func-mode) which-func-mode)
                                        (powerline-raw which-func-format nil 'l))
-                                     (powerline-raw " ")
+                                     (powerline-raw " " mode-line)
```

Comment:
i just merged a change that this will need to be updated to, i think instead of `mode-line` we're using `face0` now.